include/librealsense2/hpp/rs_processing.hpp

```diff
@@ -87,6 +87,13 @@ namespace rs2
             error::handle(e);
         }
 
+        template<class S>
+        S& operator>>(S& on_frame)
```

Comment:
I think this is more confusing than helpful. I would not have guessed that this operates on processing blocks